release-content/0.14/migration-guides/12218_Deprecate_SpriteSheetBundle_and_AtlasImageBundle.md

```diff
@@ -1,2 +1,49 @@
-- `SpriteSheetBundle` has been deprecated. Use `TextureAtlas` alongside a `SpriteBundle` instead.
-- `AtlasImageBundle` has been deprecated. Use `TextureAtlas` alongside an `ImageBundle` instead.
+`SpriteSheetBundle` has been deprecated. Insert the `TextureAtlas` component alongside a `SpriteBundle` instead.
+
+```rust
+// 0.13
+commands.spawn(SpriteSheetBundle {
+    texture,
+    atlas: TextureAtlas {
+        layout,
+        ..default()
+    },
+    ..default()
+});
+// 0.14
+commands.spawn((
+    SpriteBundle {
+        texture,
+        ..default()
+    },
+    TextureAtlas {
+        layout,
+        ..default()
+    },
+));
```

Comment:
```suggestion
// Before
commands.spawn(SpriteSheetBundle {
    texture,
    atlas: TextureAtlas {
        layout,
        ..default()
    },
    ..default()
});

// After
commands.spawn((
    SpriteBundle {
        texture,
        ..default()
    },
    TextureAtlas {
        layout,
        ..default()
    },
));
```

I've been using `// Before` and `// After` in all of my code blocks, to make the guides portable in case a PR gets pushed back to 0.15.

Reply:
We have generally been using the
```
// 0.13
// 0.14
```
style since we started doing migration guides.

Reply:
See https://github.com/bevyengine/bevy-website/pull/1268#discussion_r1621362087

Reply:
Ok, feel free to resolve this then.